src/change-detection/auto-detect-changes.spec.ts

```diff
@@ -0,0 +1,50 @@
+import { ChangeDetectorRef } from '@angular/core';
```

Comment:
In these tests, you test whether Sherlock works, but Sherlock already has 100% test coverage. I find it far more interesting to test the interop with Angular. I also want to be able to detect incompatibilities with future Angular versions. For that to work, we need to test without mocks.